homeassistant/components/sonarr/config_flow.py

```diff
@@ -72,14 +70,14 @@ def async_get_options_flow(config_entry):
         """Get the options flow for this handler."""
         return SonarrOptionsFlowHandler(config_entry)
 
-    async def async_step_reauth(self, data: dict[str, Any] | None = None) -> FlowResult:
+    async def async_step_reauth(self, data: dict[str, str] | None = None) -> FlowResult:
```

Comment:
The data parameter here is the existing config entry data which can contain integer and boolean values besides strings. I suggest we keep `Any` here for the values.
```suggestion
    async def async_step_reauth(self, data: dict[str, Any]) -> FlowResult:
```